app/components/user-menu.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  classNames: ['user-menu'],
+  classNames: ['user-menu', 'dropdown'],
```

Comment:
I generally agree but sometimes it leads to overblown class names and over-specific CSS. Already we need to do some refactoring on the CSS side because of this.